homeassistant/components/dlna_dmr/config_flow.py

```diff
@@ -27,6 +27,14 @@
 from homeassistant.core import HomeAssistant, callback
 from homeassistant.exceptions import IntegrationError
 from homeassistant.helpers import config_validation as cv, device_registry as dr
+from homeassistant.helpers.service_info.ssdp import (
```

Comment:
Note: ssdp is still being imported.
It is marked as `dependencies` in manifest.json